### PR TITLE
Cm 302 create mainenance page

### PR DESCRIPTION
**Pull request by dcmcand**
**Description of change**
Add a maintenance page to the smart hub. This will allow us to point the ttahub.ohs.acf.hhs.gov domain to this page if we ever need to do maintenance on the app. We will still be able to reach the app through the tta-smarthub-prod.app.cloud.gov route. 

This PR only sets up the infrastructure. Next I will write the script to put the app into and out of maintenance mode and document how to do it.

**How to test**
Verify the page deployed properly to the sandbox environment. You can find it at https://tta-smarthub-maintenance-page-sandbox.app.cloud.gov/.

Verify the page matches the design at tta-smarthub-prod.app.cloud.gov.

Verify that there are no obvious accessibility issues with the page.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/302

**Checklist**

- [ ] Meets issue criteria
- [ ] Code tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -33,3 +33,9 @@ applications:
       instances: ((worker_instances))
       command: yarn start:worker
       memory: ((worker_memory))
+  - name: tta-smarthub-maintenance-page-((env))
+    buildpacks:
+      - staticfile_buildpack
+    root: maintenance_page
+    force_https: "true"
```
Comment (rahearn) [inline manifest.yml:40]: I don't think we'd need `force_https` (unless that's explicitly in the cloud.gov docs) and I think you should specify the memory quota for this app. I think 512M is the default and I can't imagine the `staticfile_buildpack` would need that.

**Review by kryswisnaskas — APPROVED** [review]
Looks good!

The header style differs a little bit from the main application. We might want to adjust it at some point, but this is a lower priority.

File: maintenance_page/index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+       <span>
+        Office of Head Start TTA Smart Hub
+       </span> 
+    </header>
+    <div class="container">
+        <div class="left-div">
+        </div>
+        <main>
+            <img src="eclkc-blocks-logo.png" alt="TTA Smarthub Logo">
+            <span>The TTA Smart Hub is temporarily down.</span>
```
Comment (rahearn) [inline maintenance_page/index.html:22]: The page is missing a `h1` header. I think this span is the best option to use.

File: src/worker.js [hunk 3]
```diff
@@ -1,9 +1,13 @@
+/* eslint-disable import/first */
+require('newrelic');
+
 import {} from 'dotenv/config';
 import throng from 'throng';
 import { logger, auditLogger } from './logger';
 import { scanQueue } from './services/scanQueue';
 import processFile from './workers/files';
 
+require('newrelic');
```
Comment (rahearn) [inline src/worker.js:10]: The weird double require is showing up here, just make sure it goes away in the next merge of main.

**Review by rahearn — CHANGES_REQUESTED** [review]
Looks good - just swap out that one span for an h1 and ensure the double require goes away.

**Review by rahearn — APPROVED** [review]

**Pull request merged**